chaoscenter/web/src/api/entities/project.ts

```diff
@@ -0,0 +1,24 @@
+import type { PermissionGroup } from '@models';
+
+export enum Invitation {
+  ACCEPTED = 'Accepted',
+  DECLINED = 'Declined',
+  EXITED = 'Exited'
+}
+
+export interface Members {
+  userID: string;
+  name: string;
+  username: string;
+  email: string;
+  role: PermissionGroup;
+  invitation: Invitation;
+  joinedAt: string;
+}
+
+export interface Project {
+  projectID: string;
+  name: string;
+  members: Members[];
+  UID: string;
+}
```

Comment:
Are these two interfaced being used anywhere?

Reply:
it's always good to type them all out @hrishavjha

Reply:
Types are automatically generated from swagger file, these are ig old ones.